### PR TITLE
chore: add jest coverage for packages that are missing the settings

### PR DESCRIPTION
**Pull request by zchenwei**
#### Description of changes

The PR adds jest coverage for packages(angular, vue and ui) that are missing the settings.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/angular/jest.config.js [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+  coveragePathIgnorePatterns: [
+    // ignore coverage for top level "export", jest setup, version files
+    '<rootDir>/projects/ui-angular/src/(index|setup.jest|version).ts',
+  ],
+  coverageThreshold: {
+    global: {
+      branches: 8,
+      functions: 12,
+      lines: 28,
+      statements: 29,
```
[the remark this answers is not in the document]
Comment (zchenwei) [reply packages/angular/jest.config.js:13]: Make sense though haha, we focus more on e2e functional tests back then

File: packages/ui/jest.config.js [hunk 2]
```diff
@@ -1,4 +1,19 @@
 module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.(ts|tsx)'],
+  coveragePathIgnorePatterns: [
+    // ignore coverage for top level "export"
+    '<rootDir>/src/index.ts',
+    '<rootDir>/src/theme/types/style-dictionary.d.ts',
```
Comment (calebpollman) [inline packages/ui/jest.config.js:7]: Can we add a note explaining the `style-dictionary` ignore please?

File: packages/vue/jest.config.cjs [hunk 3]
```diff
@@ -1,4 +1,19 @@
 module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.(ts|vue)'],
+  // ignore coverage for index, version and style files
```
Comment (calebpollman) [inline packages/vue/jest.config.cjs:4]: This note doesn't really match what's below 😅

File: packages/vue/jest.config.cjs [hunk 4]
```diff
@@ -1,4 +1,19 @@
...
+  coveragePathIgnorePatterns: [
+    '<rootDir>/src/(components|composables|types)/index.ts',
```
Comment (calebpollman) [inline packages/vue/jest.config.cjs:6]: Shouldn't TS types not impact coverage by default?

**Review by calebpollman — COMMENTED** [review]
Thanks @zchenwei. Left some questions/feedback

File: packages/angular/jest.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (reesscot) [reply packages/angular/jest.config.js:13]: `branches: 8`.... 😱

File: packages/vue/jest.config.cjs (hunk 3 above)
Comment (zchenwei) [reply packages/vue/jest.config.cjs:4]: 😅 Forgot to update. Thanks for catching it!

File: packages/vue/jest.config.cjs (hunk 4 above)
Comment (zchenwei) [reply packages/vue/jest.config.cjs:6]: You are right but the type file is not pure in vue

**Review by reesscot — APPROVED** [review]

File: packages/angular/jest.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (calebpollman) [reply packages/angular/jest.config.js:13]: > `branches: 8`.... 😱

Better than `0` 😂

File: packages/vue/jest.config.cjs (hunk 4 above)
Comment (calebpollman) [reply packages/vue/jest.config.cjs:6]: Got it, thanks for researching

**Review by calebpollman — APPROVED** [review]

**Pull request merged**